src/frontend/js/components/CourseGlimpse/index.tsx

```diff
@@ -16,71 +16,115 @@ const messages = defineMessages({
     description: 'Placeholder text when the course we are glimpsing at is missing a cover image',
     id: 'components.CourseGlimpse.cover',
   },
+  organizationIconAlt: {
+    defaultMessage: 'Organization',
+    description: 'Organization logo alternative text for screen reader users',
+  },
+  codeIconAlt: {
+    defaultMessage: 'Course code',
+    description: 'Course code logo alternative text for screen reader users',
+  },
+  categoryLabel: {
+    defaultMessage: 'Category',
+    description: 'Category label text for screen reader users',
+  },
```

Comment:
Ditto, do not forget to add an `id` property for each message.
By convention we use the following pattern :
`components.<COMPONENT_NAME>.<MESSAGE_PROPERTY_KEY>`

Reply:
woops! thanks